src/components/MessageInput/MessageInputLarge.js

```diff
@@ -171,7 +172,7 @@ class MessageInputLarge extends PureComponent {
       //example: "bob, joe, and sam"
       outStr = t('{{ commaSeparatedUsers }} and {{ lastUser }} are typing...', {
         commaSeparatedUsers: arr3.slice(0, -1).join(', '),
-        lastUser: arr3.slice(-1),
+        lastUser: arr3.slice(-1)[0],
```

Comment:
```suggestion
        lastUser: arr3[arr3.length - 1],
```

Reply:
or even better if you deconstruct the array using `[firstActor, ...commaSeparatedUsers] = arr3` and remove the slice parts

Reply:
Agreed. I refactored this whole function to make it more readable. (but I think I'll make a separate PR for that, since it also includes all of the conversion to hooks, etc.)